viz.yaml

```diff
@@ -13,7 +13,25 @@ info:
     width: 250    
     height: 250    
     alttext: "some text here"
+  required-packages:
+    dataRetrieval:
+      repo: CRAN
+      version: 2.6.3
+    sp:
+      repo: CRAN
+      version: 1.2-4
+    maps:
+      repo: CRAN
+      version: 3.1.1
 fetch:
+  -
+    id: get_NM_data
```

Comment:
Does this make sense now?
the id refers to New Mexico since it has a set siteID?
The fetcher only refers to getting daily values.